src/components/ExtractedInfo.tsx

```diff
@@ -157,20 +157,20 @@ export default function ExtractedInfo({ text }: InfoProps) {
           </div>
         </div>
       </div>
-      <div>
-        <div className="flex flex-wrap justify-center">
-          {jcamp && (
+      {jcamp && (
+        <div>
+          <div className="flex flex-wrap justify-center">
             <div className="p-5 m-2 shadow sm:rounded-lg">
               <div className="text-xl font-semibold text-blue-500 border-b">
```

Comment:
`text-blue` doesn't exist in our configuration. it's `text-primary`. Same for `gray` which is `neutral` instead. See https://github.com/zakodium/components#colors

Reply:
I though that I prune those by mistake 🤦